src/main/java/core/basesyntax/utils/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.utils;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+
+        return colors[random.nextInt(colors.length)].name().toLowerCase();
+
+    }
```

Comment:
Please avoid rendudant empty lines
```suggestion
    public String getRandomColor() {
        Color[] colors = Color.values();
        return colors[random.nextInt(colors.length)].name().toLowerCase();
    }
```